editor/actions.js

```diff
@@ -28,6 +28,22 @@ export function replaceBlocks( uids, blocks ) {
 	};
 }
 
+export function insertBlock( block, after ) {
+	return {
+		type: 'INSERT_BLOCK',
+		block,
+		after,
+	};
+}
+
+export function setBlockToInsert( slug, after ) {
```

Comment:
When we hover the inserter block, we need to show the placeholder right after the selected block. I guess we don't really need the exact slug of the block to be inserted, right now, we just need to know that we're going to insert a block but I thought this information could be useful in the state.